Worker.Extensions.Sql/src/SqlTriggerAttribute.cs

```diff
@@ -28,5 +37,13 @@ public SqlTriggerAttribute(string tableName, string connectionStringSetting)
         /// Name of the table to watch for changes.
         /// </summary>
         public string TableName { get; }
+
+        /// <summary>
+        /// Name of the table used to store leases.
+        /// If not specified, the leases table name will be Leases_{FunctionId}_{TableId}
+        /// More information on how this is generated can be found here
+        /// https://github.com/Azure/azure-functions-sql-extension/blob/release/trigger/docs/TriggerBinding.md#az_funcleases_
```

Comment:
If we don't already, let's make sure we have an "update docs to remove reference to trigger branch" as a post-merge-to-main task.

Reply:
Created https://github.com/Azure/azure-functions-sql-extension/issues/894 to track.